templates/rapidsai/partials/clone_and_build_blazing.j2

```diff
@@ -0,0 +1,39 @@
+{# This partial clones the BlazingSQL repo and build and installs it into the rapids conda env. #}
```

Comment:
can this file be renamed to `clone_and_build_blazing.dockerfile.j2` to match the others?